docs/rules/metadata-spacing.md

```diff
@@ -3,6 +3,11 @@
 > ✅ The "extends": "plugin:userscripts/recommended" property in a configuration
 > file enables this rule.
 
+<!-- markdownlint-disable-next-line MD033 -->
```

Comment:
I just realized that the comments in the other documentation files were wrong (57abd584e7ad7eaed9d1c0c5f3e77d436c6c13e3). Could you fix this to `no-blanks-blockquote` instead of `MD033`?

Reply:
Done